test/Tests/src/Semantic/Case_Spec.enso

```diff
@@ -1,6 +1,7 @@
 from Standard.Base import all
 import Standard.Base.Data.Array as ArrayMod
 import Standard.Base.Data.Text as TextMod
+import Standard.Base.Data.Vector as VectorMod
```

Comment:
Not sure if this comes from this PR but I don't think we should be using `VectorMod`. It's against Enso's naming convention. Let's do `Vector_Module`, please.

```suggestion
import Standard.Base.Data.Array as Array_Module
import Standard.Base.Data.Text as Text_Module
import Standard.Base.Data.Vector as Vector_Module
```